.github/workflows/build-mac.yml

```diff
@@ -0,0 +1,114 @@
+name: Build installable mac distributions
+#on: workflow_dispatch
+on:
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build-distribution:
+#    environment: release
+    strategy:
+      matrix:
+        os: [ macos-11, macos-12 ]
+
+#    permissions:
+#      contents: write
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - uses: actions/checkout@v3.0.2
+      - name: Install Python 3
+        uses: actions/setup-python@v4.1.0
+        with:
+          cache: 'pip' # caching pip dependencies
+          python-version: 3.9
+      - run: pip install -r requirements-dev.txt
+      - run: pip install -r requirements-mac-build.txt
+      - uses: actions/checkout@v3.0.2
+        with:
+          repository: Arelle/EdgarRenderer
+          path: arelle/plugin/EdgarRenderer
+      - run: rm -rf arelle/plugin/EdgarRenderer/.git
+      - uses: actions/checkout@v3.0.2
+        with:
+          repository: xbrlus/xule
+          path: tmp
+      - run: |
+          mv tmp/plugin/validate/* arelle/plugin/validate/
+          rm -rf tmp/plugin/validate
+          mv tmp/plugin/* arelle/plugin/
+          rm -rf tmp
```

Comment:
Kind of unfortunate that the steps that are common between the linux and mac workflows don't match exactly/can't be easily reused.

Reply:
Felt out of scope for my ticket, but probably something to address eventually but we should be able to nail all this down and then reuse for all builds https://docs.github.com/en/actions/using-workflows/reusing-workflows